common/events/inductiva_api/events/schemas/tasks.py

```diff
@@ -29,6 +29,7 @@ class TaskOutputUploaded(TaskEvent):
     # Output size in bytes, it's optional because the output may not be
     # available when the event is emitted by the executer tracker
     output_size: Optional[int] = None
+    terminal: bool = True
```

Comment:
I think this parameter is unnecessary.

Reply:
Useful for cases where it is not the last event sent, but has been updated to use new_status=None.